setup.py

```diff
@@ -335,7 +335,7 @@ def get_config_var(name):
 # ==========
 setup(
     name='Cartopy',
-    version='0.14.4.dev0',
+    version='0.15.0',
```

Comment:
I agree, I will add the Iris approach.

Unfortunately this doesn't get around the issue of having to update the version switcher